src/query/execute_statement.c

```diff
@@ -14809,12 +14809,67 @@ do_replicate_statement (PARSER_CONTEXT * parser, PT_NODE * statement)
     {
```

Comment:
If you don't need codes that already exist in else part, why don't you remove them?

Reply:
I removed the unused codes.